mediacapture-streams/stream-api/mediastream/mediastream-finished-add.html

```diff
@@ -1,49 +0,0 @@
-<!doctype html>
-<html>
-<head>
-<title>Adding a track to a finished MediaStream</title>
```

Comment:
This (adding a track to an active=false stream) is now explicitly allowed by the spec. Does it make sense to add a test at the same time that verifies that one can indeed do this?